app/views/movements/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= form_with model: @movement, url: category_movements_path(@category), method: :post, local: true, html: { class: "new-entity-form" } do |f| %>
+  <div class="new-entity-header">
+    <div class ="back-arrow">
+      <%= link_to category_path(@category) do %>
+        <%= image_tag "back_arrow.png", alt: "Back Arrow", class: "back-arrow-img" %>
+      <% end %>
+    </div>
+    <h2>ADD MOVEMENT</h2>
+    <%= f.submit "ADD", class: "submit-new-mov" %>
```

Comment:
- [Optional] Please kindly validate inputs properly. Right now when I click on ADD without filling out the fields when creating a movement, I got the error in the screenshot below instead of error messages. This can create a bad user experience while using your app. Also, make sure to display validation error messages when creating a Category
![Screenshot from 2023-05-05 19-03-46](https://user-images.githubusercontent.com/63087491/236534238-ef51b0be-0658-4539-bbde-01fee969cfb4.png)